tests/test_predict.py

```diff
@@ -0,0 +1,63 @@
+import pytest
+import numpy as np
+from pathlib import Path
+
+import src.predict as pr
+import src.file_readers as rd
+
+
+def test_load_model():
+    """
+    Tests load_model function, which loads the silero TTS model.
+    """
+    model = pr.load_model()
+
+    assert str(type(model)) == "<class '<torch_package_0>.multi_acc_v3_package.TTSModelMultiAcc_v3'>"
```

Comment:
This is a bad way to test type equality. Use the `isinstance` function. Here, e.g., that would look like:
`assert isinstance(model, <whatever the hell the class is named>)`

Reply:
This appears to be a custom message, that doesn't play nicely with the `isinstance` function. For now, I am testing different aspects of the model (e.g. that is contains a select speaker), but I will look into better ways to test the class of the model in the future. Removed that line for the time being.